### PR TITLE
fix gif path

### PR DESCRIPTION
**Pull request by MaxBittker**
the image isn't being copied into the dist folder, we need to look for it in the static files folder

### CODE REVIEW

File: src/sentry/static/sentry/app/components/broadcastModal.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const ReleaseAnnouncement = ({close}) => {
         <span className="badge badge-square badge-new">NEW</span>
       </h3>
       <p>
-        <img src={mediaUrl + 'dist/release-commits-modal.gif'} />
+        <img src={mediaUrl + 'images/onboarding/release-commits-modal.gif'} />
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/components/broadcastModal.jsx]: The correct solution will be to figure out why it's not being moved into the correct folder like every other image.

**Comment by mattrobenolt** [comment]
Can you back out e1baffe4a6df976d3a0151f76b82d52177e7aa42 ? This is 100% unrelated.

File: src/sentry/static/sentry/app/components/broadcastModal.jsx [hunk 2]
```diff
@@ -4,6 +4,10 @@ import ConfigStore from '../stores/configStore';
 import ApiMixin from '../mixins/apiMixin';
 import {logAjaxError} from '../utils/logging';
 
+// this is so webpack emits the file into the dist folder. The resulting url links to a superfluous bad file generated by the plugin, which we ignore.
+// eslint-disable-next-line no-unused-vars
+const gif = require('file-loader!../../images/onboarding/release-commits-modal.gif');
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/components/broadcastModal.jsx]: what

I'm definitely deferring to @benvinegar here. This can't be the best way here.

**Review by mattrobenolt — DISMISSED** [review]
JavaScript in 2017 has become the worst thing ever. The fact that this works is mind boggling.

**Comment by mattrobenolt** [comment]
Hey @MaxBittker, turns out, your first commit was actually the most correct, proving that I'm not the one who should be reviewing these things.

Can you just force push back to 7ea4bb7f53ea0173308ba55435898547d7ffc957 ?

**Review by mattrobenolt — APPROVED** [review]

**Comment by Sija** [comment]
e1baffe4a6df976d3a0151f76b82d52177e7aa42 got lost?

**Comment by mattrobenolt** [comment]
@sija you're really insistent on this. Curious: why? Haha

I had this change pulled out since it had nothing to do with this PR and it's now sitting at https://github.com/getsentry/sentry/pull/5333

**Comment by Sija** [comment]
@mattrobenolt Call me a grammar nazi :P

**Pull request merged**